modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js

```diff
@@ -1309,7 +1309,7 @@
 			}
 
 			// Update the interface ( if paused )
-			if (!this.ignoreNextNativeEvent && this._propagateEvents && this.paused && ( mw.getConfig('EmbedPlayer.EnableIpadHTMLControls') === true )) {
+			if (!this.ignoreNextNativeEvent && this._propagateEvents && this.paused || (mw.isIpad() && ( mw.getConfig('EmbedPlayer.EnableIpadHTMLControls') === true ))) {
```

Comment:
Why OR?

Reply:
Didn't wanted to remove the `EnableIpadHTMLControls` condition since I do not know what it does.
So I added the OR condition so we can enter this condition event if the `EnableIpadHTMLControls` is set to `false` and if we are using iPad.

Do you rather I will just separate the iPad conditioning to a different line?

Reply:
As this condition should only be evaluated when we are on ipad then it's better separate it outside the if condition.
If we are not on iPad then always return true for it.